src/defense.cpp

```diff
@@ -1154,6 +1161,14 @@ std::vector<itype_id> caravan_items(caravan_category cat)
                   "ak47", "762_m87", "m4a1", "556", "savage_111f", "hk_g3",
                   "762_51", "hk_g80", "12mm", "plasma_rifle", "plasma"
               };
+
+        // Add the default magazine types for each gun.
+        for (unsigned i = 0; i < ret.size(); i++) {
```

Comment:
It probably is, I was copying the loop patterns found elsewhere (e.g. defense_game::caravan())